libfastmem/files/KOSMakefile.mk

```diff
@@ -0,0 +1,10 @@
+TARGET = libfastmem.a
+OBJS = memcpy_fast.o memset_fast.o memmove_fast.o
+KOS_CFLAGS += -Iinclude
+
+include ${KOS_PORTS}/scripts/lib.mk
+
+# Override the default %.S rule from "Makefile.rules" (we want "kos-cc" instead
+# of "kos-as").
+%.o: %.S
+	kos-cc $< -o $@
```

Comment:
This can be removed, as it has been changed in Makefile.rules.
```suggestion
```

Reply:
Thank you for the update! Removed in port and in the main library repo as well.